# Pick and Place Tray on AGV

string bin_side
geometry_msgs/Pose part_pose
geometry_msgs/Pose camera_pose
uint8 part_clr
uint8 part_type
---
bool success